.github/workflows/integration-comment.yml

```diff
@@ -66,84 +66,298 @@ jobs:
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-logstash')      || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'loki'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-loki')          || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'mongodb'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-mongodb')       || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'nats'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-nats')          || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'nginx'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-nginx')         || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'opentelemetry'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-opentelemetry') || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'postgres'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-postgres')      || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'prometheus'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-prometheus')    || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'pulsar'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-pulsar')        || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'redis'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-redis')         || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'shutdown'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-shutdown')      || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'splunk'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-splunk')        || contains(github.event.comment.body, '/ci-run-all') }}
-          - test_name: 'webhdfs'
-            if: ${{ contains(github.event.comment.body, '/ci-run-integration-webhdfs')       || contains(github.event.comment.body, '/ci-run-all') }}
+    runs-on: [linux, ubuntu-20.04-8core]
+    steps:
+      - uses: actions/checkout@v3
+
+      - run: sudo npm -g install @datadog/datadog-ci
+
+      - run: docker image prune -af ; docker container prune -f
+
+      - name: amqp
+        if: ${{ contains(github.event.comment.body, '/ci-run-integration-amqp') || contains(github.event.comment.body, '/ci-run-all') }}
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 25
+          max_attempts: 3
+          retry_on: error
+          command: cargo vdev -v int start amqp ; sleep 10 ; cargo vdev -v int test -a amqp ; cargo vdev -v int stop amqp
```

Comment:
`cargo vdev integration test <NAME>` will do the start and stop if it wasn't already running. All that is missing is to add a `sleep` step which would make total sense to add right into the YAML config and handle right in `vdev` itself.

Reply:
Initially I was using `test` , but was running into errors. Adding this sleep between starting the services for the integration, and the runner seemed to help. I suspected that had something to do with the fact that before these changes, we always had to build the runner, which meant the services were already running well before the runner started. But with these changes the runner spins up much faster.
I can try going back to `test` again though to confirm that is still happening.

Reply:
The sleep no longer proved necessary so I removed it